packages/augur-ui/src/modules/modal/containers/modal-claim-fees.ts

```diff
@@ -33,14 +34,18 @@ const mapStateToProps = (state: AppState, ownProps) => ({
   ),
   pendingQueue: state.pendingQueue || [],
   claimReportingFees: ownProps,
+  forkingInfo: state.universe.forkingInfo,
 });
 
 const mapDispatchToProps = (dispatch: ThunkDispatch<void, any, Action>) => ({
   closeModal: () => dispatch(closeModal()),
   redeemStake: (options, callback) => redeemStake(options, callback),
+  disavowMarket: marketId => disavowMarket(marketId),
 });
 
 const mergeProps = (sP: any, dP: any, oP: any) => {
+  const isForking = !!sP.forkingInfo;
+  const forkingMarket = isForking ? sP.forkingInfo.forkingMarket : undefined;
```

Comment:
Does this have to be undefined in case it's not forking?

Alternative method (which would make forkingMarket = false instead of undefined) would be:
`const forkingMarket = isForking && sP.forkingInfo.forkingMarket;`

Not sure which one you find more readable.

Reply:
forkingMarket is a string, I didn't want to use it as a boolean. isForking is a boolean